namingserver/src/main/java/org/apache/seata/namingserver/manager/NamingManager.java

```diff
@@ -152,7 +154,9 @@ public Result<String> createGroup(String namespace, String vGroup, String cluste
             LOGGER.error("no instance in cluster {}", clusterName);
             return new Result<>("301", "no instance in cluster" + clusterName);
         } else {
-            Node node = nodeList.get(0);
+            Node node =
+                nodeList.stream().filter(n -> n.getRole() == ClusterRole.LEADER || n.getRole() == ClusterRole.MEMBER)
```

Comment:
why leader and memeber can be selected?

Reply:
For a Raft cluster, you need to find the leader to create the group, whereas for a regular cluster, you need to find a member to create the group.